Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -38,10 +42,17 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager) {
         encoderCodesProperty = propManager.createPersistentProperty("Drive encoder codes per rev", 512);
         maxSpeedProperty = propManager.createPersistentProperty("Max drive motor speed (rotations per second)", 5);
 
-        pVelProp = propManager.createPersistentProperty("Drive Vel P", 2);
-        iVelprop = propManager.createPersistentProperty("Drive Vel I", 0);
-        dVelProp = propManager.createPersistentProperty("Drive Vel D", -100);
-        fVelProp = propManager.createPersistentProperty("Drive Vel F", 0);
+        pVelProp = propManager.createPersistentProperty("Drive vel P", 2);
+        iVelprop = propManager.createPersistentProperty("Drive vel I", 0);
+        dVelProp = propManager.createPersistentProperty("Drive vel D", -100);
+        fVelProp = propManager.createPersistentProperty("Drive vel F", 0);
+        
+        leftDistanceProp = propManager.createEphemeralProperty("Left distance", 0);
```

Comment:
let's rename these guys "Left encoder ticks" instead so we don't get confused about units?